src/command_modules/azure-cli-appservice/tests/test_webapp_commands.py

```diff
@@ -121,11 +121,13 @@ def test_linux_webapp_quick_create(self, resource_group, resource_group_location
         webapp_name = 'webapp-quick-linux'
         plan = 'plan-quick-linux'
         self.cmd('appservice plan create -g {} -n {} --is-linux'.format(resource_group, plan))
-        self.cmd('webapp create -g {} -n {} --plan {} -i naziml/ruby-hello'.format(resource_group, webapp_name, plan))
+        self.cmd('webapp create -g {} -n {} --plan {} -i derekbekoe/commute-frontend:0.3 --settings COMMUTE_API=https://api_app_name.azurewebsites.net'.format(resource_group, webapp_name, plan))
```

Comment:
@derekbekoe, I can't find other common docker images I can use to test this, so i am using yours for now.